### ISSUE TITLE
Multiple Instances?

### ISSUE THREAD

**Issue by JeremyRand** [rdp/screen-capture-recorder-to-video-windows-free#10]
Any chance this could support multiple instances running simultaneously, with different associated settings and DirectShow device names?  E.g. I'd like to have one DirectShow device showing part of a Firefox window, while another DirectShow device shows a video chat window.  Or maybe this is already possible and I just haven't found how?

Thanks.

**Comment by GrushBrush** [comment]
Hi, is it possible to do multiple instances now? I want to use one for the window source as input to an opencv program, then I will process that and have another output window. The processed output window I want skype to see as the source. Right now Skype can only see the original source window before processing.

**Comment by rdp** [comment]
No, but funding welcome! :)

**Comment by GrushBrush** [comment]
I hear you man. It's possible for me to just hack at the source to have multiple dll with different names and handles and registry areas? Could I make 4 simultaneous capture devices bound to 4 different HWND CLSID in this way?

**Comment by rdp** [comment]
Yeah multiple registry areas would work.  I think I may have added it in
there so it can pull from a .ini file as well...
Good luck, if you feel adventurous feel free to PR!
